cypress/e2e/02CreateNewAnalyses.cy.ts

```diff
@@ -0,0 +1,124 @@
+///<reference types="cypress" />
+
+describe('Sign In user', () => {
+  it('logs in as chris user', () => {
+    cy.visit('http://localhost:3000/login')
+    cy.get("input#pf-login-username-id").type("chris")
```

Comment:
This is ok for now, but it would be better to run your tests as the newly created fake user instead of the "chris" user.

It's best to use randomized data whereever possible instead of "default" or "well-known" values because programmers may (either accidentally or intentionally) hard-code behavior dependent on those values. This is true for the ChRIS backend: the "chris" user is hard-coded to have special permissions which no other user has, and so if you test using the "chris" user, your tests might show results which wouldn't be valid for any other user.